controllers/administration.js

```diff
@@ -1538,8 +1538,7 @@ const renderClassEdit = (req, res, next) => {
 			const mode = req.locals.mode;
 			Promise.all(promises).then(
 				([teachers, gradeLevels, currentClass]) => {
-					let schoolyears = getSelectableYears(res.locals.currentSchoolData);
-					schoolyears = schoolyears.sort((a, b) => a.startDate.localeCompare(b.startDate));
+					const schoolyears = getSelectableYears(res.locals.currentSchoolData);
```

Comment:
There is probably no easy solution to have the years sorted asc && having the correct year pre-selected on creating a new class, or? This does the trick with correct preselection but the order of the options seems quiet weird, at least to me :)

Reply:
exactly, there would be a new logic needed to implement what we actually need. right now the array is filled with the "actual year" in first place and this we use in this solution to get what we need without puting too much effort into old logic

Reply:
so the question is: can we live with the order of the years or do we want to implement new logic to have a proper order there

Reply:
i would then agree, let us live with this order. Correct preselection should probably not make the user open the options anyway